Makefile

```diff
@@ -429,6 +429,26 @@ mistral-itests: requirements .mistral-itests
 	. $(VIRTUALENV_DIR)/bin/activate; nosetests $(NOSE_OPTS) -s -v --with-coverage \
 		--cover-inclusive --cover-html st2tests/integration/mistral || exit 1;
 
+.PHONY: orchestra-itests
```

Comment:
What's the compelling reason to keep this separate from itests since this will be a formal part of st2? The requirement for st2 to be running?

Reply:
There is a separate set of unit tests for orchestra that runs w/ the rest of st2 make tests.  The integration tests is a sanity check to make sure orchestra really does work.  I kept this separate because it was easy for me to bring up launchdev and then run all the orchestra tests.  We should at some point move this into the CI.  I do like the use of python to write the integration tests though cause it gave me an easy way to code for certain condition.